package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@haventec/authenticate-web-sdk",
-  "version": "1.2.0",
+  "version": "1.3.4",
```

Comment:
I think this should be at `1.2.1` since the only change set I'm seeing from develop to master is the update in the common web sdk dependency.

Reply:
The versioning has gon a bit wild, but we can't go back